lib/tree.js

```diff
@@ -17,10 +17,10 @@ module.exports = function(rootPid, callback) {
         .slice(1)
         .map(_ => _.trim())
         .reduce((acc, line) => {
-          if (line.indexOf(rootPid + ' ') === 0) {
-            const pid = line.split(/\s+/).pop();
-            acc.push(parseInt(pid, 10));
-            rootPid = pid;
```

Comment:
For future me, it was _this_ line (23) that caused the original problem of losing track of split trees.